.github/workflows/simorgh-local-server-tests.yml

```diff
@@ -33,19 +33,29 @@ jobs:
         id: cache
         uses: actions/cache@v3
         with:
-          path: node_modules
+          path: |
+            node_modules
+            ws-nextjs-app/node_modules
           key: node-modules-${{ hashFiles('yarn.lock') }}
 
       - name: Install Node Modules
         if: steps.cache.outputs.cache-hit != 'true'
         run: ./scripts/installNodeModules.sh
 
-      - name: Build Simorgh
+      - name: Build Simorgh - Express App
         run: yarn build:local
 
-      - name: Start Simorgh Server
+      - name: Start Simorgh Server - Express App
         run: nohup node build/server.js > /dev/null 2>&1 &
 
+      - name: Build Simorgh - Next.JS App
+        working-directory: ./ws-nextjs-app
+        run: yarn build:local
+
+      - name: Start Simorgh Server - Next.JS App
+        working-directory: ./ws-nextjs-app
+        run: nohup node build/standalone/server.js > /dev/null 2>&1 &
```

Comment:
We don't technically need the nextJS app running for the purposes of these tests - but I guess it means we don't need to come back and remember to enable them?